sync/class.jetpack-sync-module-import.php

```diff
@@ -2,86 +2,153 @@
 
 class Jetpack_Sync_Module_Import extends Jetpack_Sync_Module {
 
-	private $import_end = false;
+	/**
+	 * Tracks which actions have already been synced for the import
+	 * to prevent the same event from being triggered a second time
+	 *
+	 * @var array
+	 */
+	private $synced_actions = array();
+
+	/**
+	 * A mapping of known importers to friendly names.
+	 * Keys are the class name of the known importer.
+	 * Values are the friendly name.
+	 *
+	 * @var array
+	 */
+	private static $known_importers = array(
```

Comment:
we could use [Sync Settings](https://github.com/Automattic/jetpack/blob/master/sync/class.jetpack-sync-settings.php) here, the good thing about it is that they are easy to update remotely or via cli if one day we want to add a new importer to the list without waiting for a release.

Reply:
I've updated the PR to use a default sync setting for `known_importers`, using the conventions I found in `Jetpack_Sync_Settings` and `Jetpack_Sync_Defaults`